metis-core/metis-framework-rest-client/src/main/java/eu/europeana/metis/framework/rest/client/DsOrgRestClient.java

```diff
@@ -99,7 +107,42 @@ public void deleteOrganization(Organization org) throws ServerException {
      */
     public List<Organization> getAllOrganizations() throws ServerException {
         try {
-            List<Organization> orgs =  template.getForObject(hostUrl + ORGANIZATIONS, OrganizationList.class).getOrganizations();
+
+            List<Organization> orgs =  template.getForObject(hostUrl + ORGANIZATIONS+"?apikey="+apikey, OrganizationListResponse.class).getResults();
+            return orgs;
+        } catch (Exception e) {
+            throw new ServerException("Organizations could not be retrieved with error: " + e.getMessage());
+        }
+    }
+
+    /**
+     * Retrieve all the organizations stored in METIS (OK)
+     * @return The list of all the organizations stored in METIS
+     * @throws ServerException
+     */
+    public List<Organization> getAllOrganizationsByIsoCode(String isoCode) throws ServerException {
+        try {
+
+            List<Organization> orgs =  template.getForObject(hostUrl + ORGANIZATIONS_ISOCODE+"?apikey="+apikey+"&isoCode="+isoCode, OrganizationListResponse.class).getResults();
```

Comment:
yes we can, or rather we should, not only here..on every REST API that we have.. no magic strings